specification/storagesync/resource-manager/Microsoft.StorageSync/stable/2018-07-01/storagesync.json

```diff
@@ -2267,6 +2276,153 @@
         "x-ms-long-running-operation": true
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.StorageSync/storageSyncServices/{storageSyncServiceName}/registeredServers/{serverId}/triggerRollover": {
+      "post": {
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "storageSyncServiceName",
+            "in": "path",
+            "description": "Name of Storage Sync Service resource.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "serverId",
+            "in": "path",
+            "description": "Server Id",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "description": "Body of Trigger Rollover request.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/TriggerRolloverRequest"
+            }
+          }
+        ],
+        "tags": [
+          "RegisteredServer Resource",
+          "Actions"
+        ],
+        "operationId": "RegisteredServers_triggerRollover",
+        "description": "Recall a serverendpoint.",
+        "x-ms-examples": {
+          "RegisteredServers_triggerRollover": {
+            "$ref": "./examples/RegisteredServers_TriggerRollover.json"
+          }
+        },
+        "responses": {
+          "200": {
```

Comment:
Is there a response body on the successful completion of the rollover?

Reply:
no response to this request